Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -332,17 +332,17 @@ import BraintreeDataCollector
                     return
                 }
                 
-                self.payPalContextID = approvalURL.pairingID
+                self.payPalContextID = approvalURL.baToken ?? approvalURL.ecToken
```

Comment:
If the `baToken` is null here should we fail fast and return? Or do we need to perform the risk call before failing?

Reply:
We don't need to perform the risk call but only care about the `baToken` if in the app switch flow, which we don't know yet at this point. For the web based flow the `payPalContextID` here is only used for analytics.